editor.planx.uk/src/components/Header.tsx

```diff
@@ -396,23 +396,17 @@ const EditorToolbar: React.FC<{
   headerRef: React.RefObject<HTMLElement>;
   route: Route;
 }> = ({ headerRef, route }) => {
+  const { navigate } = useNavigation();
   const [open, setOpen] = useState(false);
   const [togglePreview, user] = useStore((state) => [
     state.togglePreview,
     state.getUser(),
   ]);
 
-  const { navigate } = useNavigation();
-
   const handleClose = () => {
     setOpen(false);
   };
 
-  const handleClick = (href?: string) => {
-    if (href) navigate(href);
-    setOpen(false);
```

Comment:
This is now redundant - the dropdown closes automatically whenever you click away from it.